### DISCUSSION TITLE
Require JSON file

### DISCUSSION THREAD

**Discussion by elldur** [webdiscus/pug-plugin#70] [Q&A]
```
link(rel="manifest" href=require("/meta/webmanifest.json"))
\/
<link rel="manifest" href="{... json contents}">
```
Despite `{test: /\.(ico|svg|png|ttf|json)$/, type: "asset/resource"}` being in webpack config. Loading works correctly when importing in javascript (using import not require). The problem is fixed by using a different extension e.g. `.webmanifest`.

`require` treats json differently even outside of pug but the solutions I found involve changing `require.extensions` which does not exist in pug since require seems to be a wrapper for `mod.require` which is not accessible within pug.

Is there a way around this other than changing the extension?

**Comment by webdiscus** [comment]
Hello @elldur,

> Loading works correctly ...

I don't know what is received and what is expected exactly.
Do you can please create a small repo with reproducible problem?
